backend/coins/btc/maketx/maketx.go

```diff
@@ -142,12 +142,16 @@ func NewTxSpendAll(
 ) (*TxProposal, error) {
 	selectedOutPoints := []wire.OutPoint{}
 	inputs := []*wire.TxIn{}
+	previousOutputs := make(PreviousOutputs, len(selectedOutPoints))
```

Comment:
len(selectedOutpoints) is `0`. did you mean `len(spendableOutputs)`?

Reply:
oh yes..